snuba/datasets/errors_replacer.py

```diff
@@ -76,6 +77,9 @@ class ReplacementContext:
 
 
 class Replacement(ReplacementBase):
+
+    replacement_type: str
```

Comment:
can `replacement_type` be anything or are there a finite set of things it can be?

Reply:
I think it would only be one of these [strings](https://github.com/getsentry/snuba/blob/531147eabcc7054ab410e1c525415b919d630e4e/snuba/datasets/errors_replacer.py#L295-L335). Should this stuff be replaced with a string enum?